types/index.d.ts

```diff
@@ -0,0 +1,43 @@
+import { RequestHandler } from "express";
+import https from 'https';
+import jose from 'jose';
+
+declare namespace errors {
+    declare class InsufficientScopeError extends Error {
```

Comment:
Intellij complains about using `declare` inside `declare`, but I'm not sure if it's wrong or not.